configs/fp16/metafile.yml

```diff
@@ -8,9 +8,12 @@ Collections:
       Architecture:
         - Hard Voxelization
     Paper:
-      - https://www.mdpi.com/1424-8220/18/10/3337
-      - https://arxiv.org/abs/1812.05784
+      Title:
```

Comment:
Follow here https://github.com/open-mmlab/mmdetection/blob/master/configs/fp16/metafile.yml to update the paper/title/code of FP16.